packages/app/src/app/components/WorkspaceSelect/WorkspaceSelect.tsx

```diff
@@ -150,12 +168,16 @@ export const WorkspaceSelect: React.FC<WorkspaceSelectProps> = React.memo(
                       gap={1}
                     >
                       <Text css={{ width: '100%' }} size={3}>
-                        {team.id === state.personalWorkspaceId
-                          ? 'Personal'
-                          : team.name}
+                        {team.name}
                       </Text>
 
-                      {isTeamFree && <Badge variant="trial">Free</Badge>}
+                      {isTeamFree && !isDeactivated && (
+                        <Badge variant="trial">Free</Badge>
+                      )}
+                      {isPersonalProLegacy && <Badge variant="pro">Pro</Badge>}
+                      {isDeactivated && (
+                        <Badge variant="neutral">Deactivated</Badge>
+                      )}
```

Comment:
Wondering if we can make something like this to make it more readable
```suggestion
											{isDeactivated ? 
  											<Badge variant="neutral">Deactivated</Badge>
                       : isTeamFree ? 
                        <Badge variant="trial">Free</Badge>
                       : isPersonalProLegacy && <Badge variant="pro">Pro</Badge>}
```

Reply:
I messed it up in the last step, because we're in this intermediate state where deactivated and free can overlap. but after we get a new field on the team, each flag should be a single condition, so I'd keep it linear

```
{isA && A}
{isB && B}
{isC && C}
```

hope that makes sense